lms/static/sass/elements/_course-card.scss

```diff
@@ -1,8 +1,46 @@
-.header-img{
-  @include float(left);
-  margin-right:20px;
-}
+$course-enroll-green: rgba(15, 124, 57, 1) !default;
+$course-enroll-green-light: rgba(13, 181, 83, 1) !default;
```

Comment:
It would be preferable to use the Pattern Library colours (http://ux.edx.org/design_elements/colors/), this may be a conversation with Roderick